src/mumble/Log.cpp

```diff
@@ -345,8 +346,11 @@ void LogConfig::on_qtwMessages_itemClicked(QTreeWidgetItem *item, int column) {
 	if (item && item != allMessagesItem && column == ColStaticSoundPath) {
 		AudioOutputPtr ao = Global::get().ao;
 		if (ao) {
-			if (!ao->playSample(item->text(ColStaticSoundPath), Global::get().s.notificationVolume))
+			AudioOutputToken sample =
+				ao->playSample(item->text(ColStaticSoundPath), Global::get().s.notificationVolume);
+			if (sample == AudioOutputToken::None()) {
```

Comment:
```suggestion
			if (!ao->playSample(item->text(ColStaticSoundPath), Global::get().s.notificationVolume)) {
```
Use new implicit bool conversion